app/konnect/dev-portal/applications/dynamic-client-registration/curity.md

```diff
@@ -75,28 +75,99 @@ To get started configuring Curity, log in to your Curity dashboard and complete
+
+5. Input the **Issuer URL** of your Curity authorization server, formatted as `https://CURITY_INSTANCE_DOMAIN/oauth/v2/oauth-anonymous/.well-known/openid-configuration`
 
-3. Click the **Application Setup** tab to open the DCR settings for your Dev Portal.
+6. Select Curity as the **Provider Type**.
 
-4. Select **Curity** as the external identity provider.
+7. Enter the Client ID of the admin client created in Curity above into the **Initial Client ID** field. Enter the value you saved for the Client secret into the **Initial Client Secret** field.  Note: The Initial Client Secret will be stored in isolated, encrypted storage and will not be readable through any Konnect API.
 
-5. Enter the **Issuer URL** for your authorization server, it will look something like `https://CURITY_INSTANCE_DOMAIN/oauth/v2/oauth-anonymous/.well-known/openid-configuration`
+8. Save your DCR Provider. You should now see it in the list of DCR providers.
 
-6. If you are using the Curity configuration described in the previous sections, enter the `sub` into the **Claims** field and leave the **Scopes** field empty. If you configured Curity differently, then ensure you add the correct **Scopes** and **Claims**.
+9. Click the **Auth Strategy** tab to see all your Auth Strategies. Select **New Auth Strategy** to create an auth strategy that uses the DCR Provider you created.
 
-7. Enter the Client ID of the admin client created in Curity above into the **Initial Client ID** field.
+10. Enter a name for internal use in {{site.konnect_short_name}} and a display name that will be displayed the portal. In the **Auth Type** dropdown menu select DCR. In the **DCR Provider** dropdown, select the name of the DCR Provider config you created. Your **Issuer URL** will be prepopulated with the issuer URL you added to the DCR Provider.
 
-8. Enter the value you saved for the Client secret into the **Initial Client Secret** field.
+11. If you are using the Curity configuration described in the previous sections, enter the `sub` into the **Claims** field and leave the **Scopes** field empty. If you configured Curity differently, then ensure you add the correct **Scopes** and **Claims**.
 
-9. Click **Save**.
+12. Select the relevant **Auth Methods** you need (`client_credentials`, bearer, session) and **Save**. 
 
-   If you previously configured any DCR settings, this will
-   overwrite them.
+{% endnavtab %}
+{% navtab API %}
+After configuring Curity, you can integrate it with the Dev Portal for dynamic client registration (DCR). This process involves two steps: creating the DCR provider and establishing the authentication strategy. DCR providers are designed to be reusable configurations. This means once you've configured the Curity DCR provider, it can be utilized across multiple authentication strategies without needing to be set up again.
```

Comment:
```suggestion
After configuring Curity, you can integrate it with the Dev Portal for dynamic client registration (DCR). This process involves two steps: creating the DCR provider and establishing the authentication strategy. DCR providers are designed to be reusable configurations. This means once you've configured the Curity DCR provider, it can be used across multiple authentication strategies without needing to be set up again.
```